### ISSUE TITLE
MongoDB 3.2: support new expression and accumulation operators

### ISSUE THREAD

**Issue by mossprescott** [quasar-analytics/quasar#1311]
Depends on #1308.

New for `AccumF`:
- [ ] [`$stdDevSamp`](https://docs.mongodb.com/manual/reference/operator/aggregation/stdDevSamp/#grp._S_stdDevSamp) and [`$stdDevPop`](https://docs.mongodb.com/manual/reference/operator/aggregation/stdDevPop/#grp._S_stdDevPop)

New for `Expr`:
- [ ] `$sqrt`, `$abs`, `$log`, `$log10`, `$ln`, `$pow`, `$exp`, `$trunc`, `$ceil`, `$floor` will each come with a new standard library function
- NB: support for new array operators `$slice`, `$arrayElementAt`, `$concatArrays`, `$isArray`, `$filter` will be tracked a separate issue
- NB: a subset of `Accum` ops can now be used directly in `$project`. Using them to avoid `$group`/`$unwind` will be tracked in a separate issue.

**Comment by zeeiyerWork** [comment]
This ticket can be closed since
- Arithmetic Operators are covered in #2272
- Array Operators in #2379
- Accum operators in
